docs/config.json

```diff
@@ -14,6 +14,20 @@
       "title": "How to learn Python",
       "blurb": "An overview of how to get started from scratch with Python."
     },
+    {
+      "uuid": "7a2e1a4f-1fa8-4327-b700-5af101fcdc89",
+      "slug": "test-driven-development",
+      "path": "docs/TDD.md",
+      "title": "How to use Exercism tests",
```

Comment:
```suggestion
      "title": "Test Driven Development",
```

Reply:
We have an important document on how to set up pytest and test solutions for Python and we don't want to create confusion with this title.